### PR TITLE
DESKTOP VERSION REVIEW REQUEST

### PR DESCRIPTION
**Pull request by pmaxy**
Things i did:

> Add new CSS file.
> Use media query.
> Add styles

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -19,6 +19,7 @@ body {
   margin: auto;
   background: #f7f7f9;
   position: relative;
+  height: 5500px;
```
Comment (Felix45) [inline style.css]: - [ Required ] Please note that for mobile devices with a width greater than `375px` the width of the content on the webpage is not responsive. Your design maintains a width of `375px` even when the device has a width greater than `375px`. It's good practice to not apply fixed widths to elements on your page in order to make the webpage responsive, for example, to make the mobile navigation menu responsive across all mobile devices i would recommend the following `CSS` rules.

```CSS
body {
  font-family: 'Poppins', sans-serif;
  align-items: center;
  margin: auto;
  background: #f7f7f9;
  position: relative;
  height: 5500px;
}

header {
  left: 0;
  top: 0;
  right: 0;
  height: 92px;
  display: flex;
  flex-direction: row;
  justify-content: space-between;
  align-items: center;
  position: sticky;
  z-index: 2;
  background-color: #fff;
  margin: 0;
}
```
Removing the fixed width on the `body` and `header` elements makes the navigation menu adapt to mobile screen sizes with widths greater than `375px`
- Please consider dropping all the fixed widths on different sections of your design for mobile devices, please see the screenshot below for more details on this. The margins on the design at a width of `576px` should not be this large.

Screenshot from 2022-12-08 15-37-25

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @pmaxy , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights :heavy_check_mark: 
- [ ] All the linter checks are setup and passing with green checks :white_check_mark: 
- [ ] GitHub flow is used correctly :white_check_mark: 

almost-there

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: desktop.css [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+    width: auto;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+    height: auto;
+    padding-right: 500px;
+    padding-bottom: 12px;
+    overflow: none;
+  }
```
Comment (franclobo) [inline desktop.css]: - [ ] **[OPTIONAL]** On the desktop version there are two lines in the Figma version but in yours, there are three, please 🙏🏽 correct it to be similar to the Figma version.

| Figma version | Your version |
|---|---|
| image | image |

**Review by franclobo — APPROVED** [review]
Hi @pmaxy, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 
- Professional README file ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**